### PR TITLE
Summary page improvements

### PR DESCRIPTION
**Pull request by karpov-sv**
*a.k.a. New Year PR*

Set of visual fixes, adjustments and improvements for more consistent experience on the summary page and around it.

Based on #541 and thus has unnecessarily long commit list shown. Hope it will be reduced after #541 merge.

As usual, demo version is available at http://157.136.250.93:24000

### CODE REVIEW

File: apps/plotting.py [hunk 1]
```diff
@@ -1919,31 +2095,31 @@ def draw_cutout(data, title, lower_bound=0, upper_bound=1, modal=False, zoom=Tru
...
+        figure=fig,
+        style=style,
+        config={'displayModeBar': False},
+        className=classname,
+        responsive=True
+    )
 
     return graph
 
+app.callback([
```
Comment (JulienPeloton) [inline apps/plotting.py:2113]: You forgot `@` before the decorator?

File: apps/cards.py [hunk 2]
```diff
@@ -956,15 +908,21 @@ def card_id1(object_data, object_uppervalid, object_upper):
...
-                ```python
-                Discovery date: {}
-                Last detection: {}
-                Number of detections: {}
-                Number of low quality alerts: {}
-                Number of upper limits: {}
-                ```
+                Discovery date: `{}`
+                Last detection: `{}`
+                Duration: `{:.2f}` / `{:.2f}` days
```
Comment (JulienPeloton) [inline apps/cards.py:913]: The redesign is great! However, I could not understand what the 2 numbers in `Duration: 15.94 / 19.86 days` mean? g & r duration?

**Review by JulienPeloton — COMMENTED** [review]
Thanks -- there are many super improvements! I leave here my feedback on suggested changes.

## dmc.Paper without options

You systematically removed the options `radius='xl', p='md', shadow='xl', withBorder=True` to `dmc.Paper` component for lightcurves. I prefer by far to keep the options as before, as rendering without the options is not great to my opinion.

## Summary page accordion (right side)

I noticed an opened item does not close anymore when I open a new one. Is it on purpose? I cannot find the option on the code that does that (and I prefer when only one item at a time is open -- i.e. the previous behaviour).

## Stamp model removed

Some people reported they were using the enlarged stamps from the modal because the ones on the right were too small. I will be against making the current cutouts bigger for space reasons, but probably we should reintroduce the modal with enlarged stamps?

File: apps/plotting.py (hunk 1 above)
Comment (karpov-sv) [reply apps/plotting.py]: Yes, thanks for reporting! Fixed it now

File: apps/cards.py (hunk 2 above)
Comment (karpov-sv) [reply apps/cards.py:913]: Second one is the difference between `jdendhist` and `jdstarthist` - so includes the points potentially not present in Fink, but probably important for understanding the long-term behaviour

**Comment by karpov-sv** [comment]
> Stamp modal removed

Yes, I understand the wish to have them bigger - but in its current form the popup is ugly and useless. Until we get some better idea what to add there - I suppose the zoom feature of smaller cutouts is enough for inspecting the fine details.

**Comment by karpov-sv** [comment]
> Summary page accordion (right side)

Having several panels open there is essential as otherwise you cannot simultaneously see the cutouts and alert contents for selected point - and it is important when clicking through the data.

Actually, I thought you intentionally used `AccordionMultiple` there, and just forgot to properly enable it (the befaviour is actually defined by whether you use single value or list as initial value for the component).

**Comment by JulienPeloton** [comment]
> > Stamp modal removed
> 
> Yes, I understand the wish to have them bigger - but in its current form the popup is ugly and useless. Until we get some better idea what to add there - I suppose the zoom feature of smaller cutouts is enough for inspecting the fine details.

I propose that until we get some better idea, we keep the previous button.

File: index.py [hunk 3]
```diff
@@ -502,52 +502,52 @@ def update_suggestions(n_intervals, n_submit, n_clicks, value):
...
-#     Output('search_bar_input', 'value'),
-#     Input({'type': 'search_bar_quick_field', 'index': ALL}, 'n_clicks'),
-#     State({'type': 'search_bar_quick_field', 'index': ALL}, 'children'),
-#     State('search_bar_input', 'value'),
-#     prevent_initial_call=True
-# )
-
-# TODO: convert to clientside callback, if pattern matching is supported for them
-@app.callback(
+clientside_callback(
```
Comment (JulienPeloton) [inline index.py:527]: The quickfields do not work anymore. They are clickable, but the searchbar is not updated.

**Review by JulienPeloton — APPROVED** [review]
It looks good to me -- can I merge these changes?

**Comment by karpov-sv** [comment]
Hi @JulienPeloton, yes, sure! You may merge it

**Pull request merged**